Anlab.Mvc/Views/TestItems/Index.cshtml

```diff
@@ -81,17 +88,35 @@
     @{ await Html.RenderPartialAsync("_DataTableScriptsPartial"); }
     <script type="text/javascript">
         $(function() {
-            $("#table").dataTable({
+            var table = $("#table").DataTable({
```

Comment:
@laholstege does this work even w/ upper-cased DataTable()?  I would think it would need to be .dataTable().  But maybe they aliased it